actors/miner/src/emit.rs

```diff
@@ -16,11 +17,13 @@ pub fn sector_activated(
     sector: SectorNumber,
     unsealed_cid: Option<Cid>,
     pieces: &[(Cid, u64)],
+    expiration: &ChainEpoch,
```

Comment:
Don't pass integers by reference. This goes for all the ChainEpochs, ActorIDs  etc below e.g in `claim.rs`. Just move/copy.

Reply:
Removed this as we're not adding the expiration event to the Sector events anymore.